source/02-EV/01.ptx

```diff
@@ -12,14 +12,21 @@
 
 <subsection><title>Class Activities</title>
 
+<aside>
+  <p>We will use the phrase <term>vector space</term> freely from this point on, even while delaying a formal definition.
+    Readers can choose to interpret this to mean <em>Euclidean vector space</em>, i.e <m>\IR^n</m> for some <m>n</m>, if they wish;
+    we do this as all of the statements we make using the term <term>vector space</term> are also true for all vector spaces
+    as defined in <xref ref="AT5-properties"/>.
+    </p>
+</aside>
```

Comment:
Go ahead and tell instructors (and students) that we will use the word "vector space" before formally defining it.  As an `<aside>`, this will not display in the slides.